jib-core/src/main/java/com/google/cloud/tools/jib/frontend/MainClassFinder.java

```diff
@@ -61,7 +61,7 @@ public static String resolveMainClass(
       throws MainClassInferenceException {
     BuildLogger logger = projectProperties.getLogger();
     if (mainClass == null) {
-      logger.info(
+      logger.lifecycle(
```

Comment:
Let's actually hold off on this line for now, since this may be confusing for many first-time users.